test/bar-polar-stack.html

```diff
@@ -91,5 +104,205 @@
                 window.onresize = chart.resize;
             });
         </script>
+        <script>
+            require([
+                'echarts'
+            ], function (echarts) {
+
+                var chart = echarts.init(document.getElementById('polar-negative'), null, {});
+
+                var data = [];
+
+                chart.setOption({
+                    angleAxis: {
+                    type: 'category',
+                    data: ['S1', 'S2', 'S3']
+                },
+                radiusAxis: {
+                    min:-1,
+                    max: 3
+                },
+                polar: {
```

Comment:
Try adding `radius: [50, 200]` to all `polar` in the test cases to find potential bugs.